drivers/sensor/lsm6dsl/lsm6dsl.h

```diff
@@ -689,12 +688,24 @@
 #endif
 
 struct lsm6dsl_config {
-	char *i2c_master_dev_name;
-	u16_t i2c_slave_addr;
+	char *comm_master_dev_name;
+};
+
+struct lsm6dsl_data;
+
+struct lsm6dsl_transfer_function {
+	int (*read_data)(struct lsm6dsl_data *data, u8_t reg_addr,
+			u8_t *value, u8_t len);
```

Comment:
Align on upper line struct, apply to following lines

Reply:
OK, I'll change all the things highlighted.
Did you check the (*write_data) implementation especially the SPI one?
I tested it on ArgonKey and should be ok, but would be nice to give a loom at it.

Reply:
Tested ok on disco_l475_iot1